cypher/cypher-docs/src/docs/dev/query-tuning.asciidoc

```diff
@@ -57,6 +89,30 @@ When Cypher is building execution plans, it looks at the schema to see if it can
 These index decisions are only valid until the schema changes, so adding or removing indexes leads to the execution plan cache being flushed.
 
 
+[[cypher-preparser-runtime]]
+=== Cypher runtime
+
+Using the execution plan, the query is executed -- and records returned -- by the query engine, or _runtime_.
+Depending on whether Neo4j Enterprise Edition or Neo4j Community Edition is used, there are two different runtimes available:
+
+Interpreted::
+In this runtime, the operators in the execution plan are chained together in a tree, where each non-leaf operator feeds from one or two child operators.
+The tree thus comprises nested iterators, and the records are streamed in a pipelined manner from the top iterator, which pulls from the next iterator and so on.
+
+Compiled::
+Algorithms are employed to group intelligently the operators in the execution plan in order to generate new combinations and orders of execution which are optimised for performance and memory usage.
+While this should lead to superior performance in most cases, it is still under development.
```

Comment:
and does not support all possible operators or queries.

Reply:
👍